Slambook/index.js

```diff
@@ -1,14 +1,26 @@
 let form;
 
```

Comment:
There is no need to declare an empty 'err' variable at line 2 since it is being declared again later in the code at line 24. You can remove line 2 to avoid confusion.